docs/source/custom_resolvers.rst

```diff
@@ -391,3 +391,71 @@ as interpolations), and they return a ``ListConfig`` that contains keys or value
     >>> show(cfg.ips)
     type: ListConfig, value: ['${workers.node3}', '${workers.node7}']
     >>> assert cfg.ips == ["10.0.0.2", "10.0.0.9"]
+
+.. _clearing_resolvers:
+
+Clearing/removing resolvers
+---------------------------
+
+.. _clear_resolvers:
+
+clear_resolvers
+^^^^^^^^^^^^^^^
+
+Clearing (removing) all custom resolvers at once is supported using ``OmegaConf.clear_resolvers()``.
+This method removes all custom resolvers, but retains all the default resolvers.
+
+.. code-block:: python
+
+    def clear_resolvers() -> None
+
+In the following example, first we register a new custom resolver ``str.lower``, and then clear all
+custom resolvers.
+
+.. doctest::
+
+    >>> # register a new resolver: str.lower
+    >>> OmegaConf.register_new_resolver(
+    ...     name='str.lower',
+    ...     resolver=lambda x: str(x).lower(),
+    ... )
+    >>> # check if resolver exists (after adding, before removal)
+    >>> OmegaConf.has_resolver("str.lower")
+    True
+    >>> # clear all custom-resolvers
+    >>> OmegaConf.clear_resolvers()
+    >>> # check if resolver exists (after removal)
+    >>> OmegaConf.has_resolver("str.lower")
+    False
+    >>> # default resolvers are not affected
+    >>> OmegaConf.has_resolver("oc.env")
+    True
```

Comment:
Granted it's down right simple :wink: . But, I believe, examples are kind of a major useful feature of the documentation of this library. So, I thought if it is made clear through an example, it will be helpful. I wanted to show that custom resolvers are removed, while default ones are not, when `clear_resolvers()` is used.

However, if you insist on it's (the example: starting from line-412) removal, that's okay as well.